packages/common/container-definitions/package.json

```diff
@@ -62,6 +62,19 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"RemovedEnumDeclaration_ContainerErrorType": {
```

Comment:
These should not longer be broken, right?

Reply:
Based on our discussion, we seem to have confused our type-tests. Intellisense is showing that the types of the properties in question are the same string literals they were before. And consumers of the interfaces (including those which set the values in `container-utils`) build just fine. So this seems like a type-test issue and not a real incompatibility.

Reply:
Proof from intellisense that it evaluates to the same string literal!
<img width="729" alt="Screenshot 2023-08-16 at 4 12 59 PM" src="https://github.com/microsoft/FluidFramework/assets/6777404/9e6f0518-490f-4b75-ad88-56adb7f46e98">